modules/core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/event/test/WorkTest.java

```diff
@@ -60,6 +63,16 @@
     @Inject
     protected CoreSession session;
 
+    @Inject
+    protected WorkManagerFeature workManagerFeature;
+
+    // TODO NXP-28931: remove
+    @Before
```

Comment:
```suggestion
    @BeforeClass
```
?

Reply:
I can't because I'm using the non static member `workManagerFeature`, if I'm not mistaken.

Reply:
Ha... right.